go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/Azure/azure-sdk-for-go v63.0.0+incompatible
 	github.com/Azure/go-autorest/autorest v0.11.25
 	github.com/Azure/go-autorest/autorest/adal v0.9.18
-	github.com/GoogleCloudPlatform/prometheus-engine v0.4.1
+	github.com/GoogleCloudPlatform/prometheus-engine v0.5.1-0.20221123211114-163beef5eba8
```

Comment:
The main reason I can think of for depending on a tagged version is to ensure the user-agent injects the proper prometheus-engine export library used. We should probably wait to tag it there before doing the final cut here.

Reply:
@realschwa can you bump this to point to https://github.com/GoogleCloudPlatform/prometheus-engine/releases/tag/v0.6.0-rc.0?

Reply:
@pintohutch done